entrypoint.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z "${API_CREDENTIALS}" ]]; then
+  echo -e "\e[31mAPI_CREDENTIALS needs to be set in env. Exiting.\e[0m"
+  exit 1
+fi
+
+#
+# Get label and see if it is a changelog label.
+# If it isn't a changelog label, let's exit.
+#
+
+CHANGELOG_LABEL=$(
+  jq -r '.label.name' ${GITHUB_EVENT_PATH} |
+  grep 'changelog - ' |
+  grep -o -E 'added|changed|fixed' ||
+  true
+)
+
+if [ -z ${CHANGELOG_LABEL} ];
+then
+  echo -e "\e[34mLabel isn't a changelog label. Exiting.\e[0m"
```

Comment:
Maybe log the label here? In case this changes in the future, eg. for some reason we add one of [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)'s currently unused labels (deprecated, removed, security), and this needs to be debugged.

Not a terrific reasoning but I'd rather be safe than sorry.

Reply:
done